src/3rd_party/phf/phf.cc

```diff
@@ -679,7 +679,7 @@ PHF_PUBLIC void PHF::compact(struct phf *phf) {
 	}
 
 	/* simply keep old array if realloc fails */
-	if ((tmp = realloc(phf->g, phf->r * size)))
+	if ((tmp = realloc(phf->g, phf->r * size)) != 0)
```

Comment:
3rd party code, but maybe let's turn this into two lines. I am allergic to assignments inside if-conditions.

Reply:
Changed.